src/components/calcite-pick-list-item/calcite-pick-list-item.tsx

```diff
@@ -49,6 +49,11 @@ export class CalcitePickListItem {
    * When false, the item cannot be deselected by user interaction.
    */
   @Prop() disableDeselect = false;
+  
+  /**
+   * When true, the item cannot be selected by user interaction.
+   */
+  @Prop({ reflect: true }) disableSelect = false;
```

Comment:
The name of this prop is a bit confusing. Right above it we have `disableDeselect` and this is `disableSelect`. It sounds like they would conflict? Any way these can be merged or named differently?

I thought for naming we had proposed names like `selectDisabled`?

Reply:
Actually, thinking about it now, we should probably use what we used in ListItem: `nonInteractive`. Yeah?

Reply:
yeah

Reply:
@driskull Updated.